src/algorithms.jl

```diff
@@ -0,0 +1,9 @@
+abstract type SciMLNLSolveAlgorithm end
+
+struct JuliaNLsolve <: SciMLNLSolveAlgorithm
+

```

Comment:
It should have method choices: https://github.com/JuliaNLSolvers/NLsolve.jl#fine-tunings

Reply:
And an autodiff choice: https://github.com/JuliaNLSolvers/NLsolve.jl#automatic-differentiation

Reply:
Done